xwiki-platform-core/xwiki-platform-eventstream/xwiki-platform-eventstream-api/src/main/java/org/xwiki/eventstream/query/CompareQueryCondition.java

```diff
@@ -78,7 +78,14 @@ public class CompareQueryCondition extends AbstractPropertyQueryCondition
          * 
          * @since 14.0RC1
          */
-        ENDS_WITH
+        ENDS_WITH,
+
+        /**
+         * The property value contains the passed value.
+         *
+         * @since 14.4RC1
+         */
+        CONTAINS
```

Comment:
Shouldn't it be marked as unstable?

Reply:
Good point, now I'm wondering why the other two recently added constants aren't marked as unstable.

Reply:
> I'm wondering why the other two recently added constants aren't marked as unstable

I will go for the ravages of aging :)